cmd/store/import.go

```diff
@@ -116,30 +117,73 @@ func importStore(
 	maxTuplesPerWrite, maxParallelRequests int32,
 	fileName string,
 ) (*CreateStoreAndModelResponse, error) {
-	var (
-		response *CreateStoreAndModelResponse
-		err      error
-	)
+	response, err := createOrUpdateStore(clientConfig, fgaClient, storeData, format, storeID, fileName)
+	if err != nil {
+		return nil, err
+	}
 
+	if len(storeData.Tuples) == 0 {
+		fmt.Println("No tuples to import.")
```

Comment:
We've had issues previously (#354) with folks relying on this command producing JSON parseable output. I think we should remove this log to avoid breaking that again.